conn_linux.go

```diff
@@ -165,6 +171,8 @@ func (c *conn) Receive() ([]Message, error) {
 		return nil, errInvalidFamily
 	}
 
+	n = nlmAlignOf(n)
```

Comment:
Can just use nlmsgAlign defined in align.go and remove the new code.

Reply:
sorry that I missed this existing function before.